### PR TITLE
[Post] ES6 classes

### PR DESCRIPTION
**Pull request by magsout**

### CODE REVIEW

File: content/fr/articles/js/es2015/classes.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+// appel de la méthode sayName()
+console.log(user.sayName()) // John Doe
+```
+
+Pour rappelle, voici une [façon possible
+d'écrire](https://gist.github.com/magsout/a876b2fa8240a987e523) cette classe en
+`es5` :
+
+```js
+var User = function(firstname, lastname) {
```
Comment (naholyr) [inline content/fr/articles/js/es2015/classes.md]: To be fair, this test is missing:

```js
if (!(this instanceof arguments.callee)) {
  throw new TypeError("Class constructors cannot be invoked without 'new'")
}
```

**Comment by naholyr** [comment]
Maybe a few things missing:

* ``class``es cannot be called as functions, they **require** the ``new`` operator (unlike simple functions, cf. my commit comment)
* you can call a parent method with ``super.method()``
* just like a ``function``, a ``class`` can be used as an expression, you can even omit the name if you wish : ``var Classs = class { … }``
* ``new.target`` is the original object in the chain of constructors, but that may be too advanced?

File: content/fr/articles/js/es2015/classes.md (hunk 1 above)
Comment (bloodyowl) [inline content/fr/articles/js/es2015/classes.md]: as `arguments.callee` isn't present in strict mode, that could be a bit confusing.

Comment (bloodyowl) [inline content/fr/articles/js/es2015/classes.md]: using `function User() {` and `instanceof User` would be simpler

File: content/fr/articles/js/es2015/classes.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+  this.firstname = firstname
+  this.lastname = lastname
+}
+
+User.prototype.sayName = function() {
+  return(this.firstname + " " + this.lastname)
```
Comment (bloodyowl) [inline content/fr/articles/js/es2015/classes.md]: missing space

**Comment by bloodyowl** [comment]
agreed with @naholyr small list

**Comment by magsout** [comment]
thanks for the review @bloodyowl  and @naholyr 

If you want to make a PR .. (no time to review my post before tomorow)

**Comment by bloodyowl** [comment]
are you ok if I fix it in your PR?

**Comment by magsout** [comment]
@bloodyowl yes of course

**Comment by bloodyowl** [comment]
added a few fixes

**Pull request merged**